Common/Vulkan/VulkanDebug.cpp

```diff
@@ -103,3 +103,60 @@ VkBool32 VKAPI_CALL Vulkan_Dbg(VkDebugReportFlagsEXT msgFlags, VkDebugReportObje
 	// keep that behavior here.
 	return false;
 }
+
+VkBool32 VKAPI_CALL VulkanDebugUtilsCallback(
+	VkDebugUtilsMessageSeverityFlagBitsEXT           messageSeverity,
+	VkDebugUtilsMessageTypeFlagsEXT                  messageType,
+	const VkDebugUtilsMessengerCallbackDataEXT*      pCallbackData,
+	void*                                            pUserData) {
+	const VulkanLogOptions *options = (const VulkanLogOptions *)pUserData;
+	std::ostringstream message;
+
+	const char *pMessage = pCallbackData->pMessage;
+	int messageCode = pCallbackData->messageIdNumber;
+	const char *pLayerPrefix = "";
+	// Ignore perf warnings for now. Could log them, so still want them registered.
```

Comment:
Hm.  Is it ignoring them?  Maybe this belongs above line 150?

-[Unknown]

Reply:
Oops!